client/signup/p2-step-wrapper/style-p2v2.scss

```diff
@@ -0,0 +1,158 @@
+@import url( 'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;900&display=swap' );
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+@import 'calypso/assets/stylesheets/_p2-extends.scss';
+
+.is-p2v2 {
```

Comment:
I'm not sure if we could somehow optimize these class names so that we don't have to put everything under `.is-p2v2`. Or maybe it's okay? I guess since we are reusing so much (ie p2-step-wrapper) but want custom styling for the v2.

Reply:
Correct, it's just to ensure I'm not affecting any of the current styles. The `.is-p2v2` wrapper is temporary and will be removed entirely when the revised flow becomes the main flow. `style-p2v2.scss` will become `style.scss` as `p2v2` becomes `p2`.